v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/transforms/AssignShardIdFn.java

```diff
@@ -337,6 +337,11 @@ private Map<String, Object> fetchSpannerRecord(
    */
   private void marshalSpannerValues(
       ObjectNode newValuesJsonNode, String tableName, String colName, Struct row) {
+    if (row.isNull(colName)) {
```

Comment:
Is this handled in scenarios where the custom transform returns the shard id field as well ?